ubuntu-core-image/README.md

```diff
@@ -68,18 +78,28 @@ Accept the device onto your Landscape.
 Wait until the computer info shows and then go to the **Snaps** tab.
 Here you can click on **Install snaps**.
 Enter snap name `ubuntu-frame` and `24/stable` for _tracking channnel_.
-Clicvk **+** and also add `wpe-webkit-mir-kiosk` and `22/candidate`.
+Click **+** and also add `wpe-webkit-mir-kiosk` and `22/candidate`.
 Click **install**.
 
 ![install snap](../media/landscape-install.png)
 
+This is equivalent to running the following on the device:
+```
+sudo snap install ubuntu-frame --channel=24/stable
+sudo snap install wpe-webkit-mir-kiosk --channel=22/candidate
+```sudo
+
 The installation command will take some time to be delivered to the device, and then executed.
 The result will be shown on Landscape.
 
 If the installation was successful, you should see the annotated camera feed being displayed on the local display connected to the Raspberry Pi.
 
 ![local monitor](../media/loca-monitor-frame.jpg)
 
-The URL that is displayed by the `wpe-webkit-mir-kiosk` is set to be the `tf-ubuntu-frame-example` endpoint by default, in the [gadget.yaml](additions-to-gadget.yaml).
+The URL that is served by the `wpe-webkit-mir-kiosk` is set to the `tf-ubuntu-frame-example`'s web server by default, in the [gadget.yaml](additions-to-gadget.yaml).
```

Comment:
I'm not sure `served` is the correct word to use here, as the URL is served by the python script. The kiosk browser visits that URL, and displays it on the screen.